Cargo.toml

```diff
@@ -1,8 +1,9 @@
 [package]
-name = "spotifycli"
+name = "crabby"
 version = "0.1.0"
 edition = "2018"
```

Comment:
Maybe we could change the edition to 2021? As this project is yet so in its inception.